README.md

```diff
@@ -1,70 +1,24 @@
-# Getting Started with Create React App
+#Styling Notes
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<img src="src/icons/styleMock.png" alt="style mock" width="600">
+Color Pallet HSL:
 
-## Available Scripts
+hsl(105, 30%, 68%)
```

Comment:
Just curious, what made you decide to go with HSL over #hex?

Reply:
I personally like hsl because it alows we to quickly add in a fourth opacity value or tweak colors based on how I want them to relate to one another: needing higher contrast, wanting a similar color but more saturated, or scewing in a different direction as far as hue.